.travis.yml

```diff
@@ -25,7 +25,7 @@ cache:
 
 before_script:
   - phpenv config-add .travis.php.ini
-  - composer self-update
+  - composer self-update 1.10.17
```

Comment:
To prevent a hard version pin, I'd prefer to see: `composer self-update --1` I only just found out about this flag, but this might be a good way to install Composer 1 in the other projects. 

Not that big sec risk on Travis tho..

Reply:
Great solution, thanks!